dotnetv3/Lambda/README.md

```diff
@@ -0,0 +1,48 @@
+# AWS Lambda code examples for .NET
+
+## Overview
+
+The examples in this section show how to use the AWS SDK for .NET with AWS Lambda to manage and invoke Lambda functions.
+
+AWS Lambda is a serverless, event-driven compute service that lets you run code for virtually any type of application or backend service without provisioning or managing servers. You can trigger Lambda from over 200 AWS services and software as a service (SaaS) applications, and only pay for what you use.
```

Comment:
With AWS Lambda, you can run code without provisioning or managing servers. You pay only for the compute time that you consume—there's no charge when your code isn't running. You can run code for virtually any type of application or backend service—all with zero administration.

Reply:
Fixed.